pkg/kubelet/kubelet.go

```diff
@@ -858,7 +858,7 @@ func (kl *Kubelet) killContainerByID(ID string) error {
 }
 
 const (
-	PodInfraContainerImage = "kubernetes/pause:latest"
+	PodInfraContainerImage = "gcr.io/google_containers/pause:0.8.0"
```

Comment:
pause image was built and pushed through build/pause/Makefile. I did this yesterday.

With this change, I just did a test

```
root@kubernetes-minion-4ztj:/var/log# docker ps 
CONTAINER ID        IMAGE                                                COMMAND                CREATED             STATUS              PORTS               NAMES
843d8ba08368        gcr.io/google_containers/pause:0.8.0                 "/pause"               5 minutes ago       Up 5 minutes                            k8s_POD.1a1fe303_valid-pod_default_f70a30fc-d8c3-11e4-8e5e-42010af00363_9ec7e923                                            
```